src/v/security/krb5_configurator.cc

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2023 Redpanda Data, Inc.
+ *
+ * Licensed as a Redpanda Enterprise file under the Redpanda Community
+ * License (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * https://github.com/redpanda-data/redpanda/blob/master/licenses/rcl.md
+ */
+
+#include "security/krb5_configurator.h"
+
+#include "config/property.h"
+
+namespace security::krb5 {
+
+configurator::configurator(config::binding<ss::sstring>&& krb5_config)
```

Comment:
in the debug build on disk we pay 1.7 MB to keep these 10 lines of code in their own .cc file. Any reason not to put this in the same file with the other krb stuff?

Reply:
The thought process was that I wanted a new header file since there was no need to bring the krb headers into kafka/server.h. Since there was a new header file, I created a new implementation file.

Is the suggestion to move just the implementation to krb5.cc? I'm happy to do that.

Reply:
yeh i think we can make multiple headers that are view onto the security library to optimize for different cases.

but i wouldn't submit a pr just to change this. more about going forward it's a small win for compilation time.